src/jumpstart/jumpstartLinkHandler.ts

```diff
@@ -1,25 +1,24 @@
 import { Contract } from '@celo/connect'
 import { ContractKit, newKitFromWeb3 } from '@celo/contractkit'
 import walletJumpstart from 'src/abis/IWalletJumpstart'
-import { getDynamicConfigParams } from 'src/statsig'
-import { DynamicConfigs } from 'src/statsig/constants'
-import { StatsigDynamicConfigs } from 'src/statsig/types'
+import { NetworkId } from 'src/transactions/types'
 import Logger from 'src/utils/Logger'
 import { fetchWithTimeout } from 'src/utils/fetchWithTimeout'
 import { getWeb3Async } from 'src/web3/contracts'
 import networkConfig from 'src/web3/networkConfig'
 import { getContract } from 'src/web3/utils'
+import { Hash } from 'viem'
 
 const TAG = 'WalletJumpstart'
 
-export async function jumpstartLinkHandler(privateKey: string, userAddress: string) {
-  const contractAddress = getDynamicConfigParams(
-    DynamicConfigs[StatsigDynamicConfigs.WALLET_JUMPSTART_CONFIG]
-  ).jumpstartContracts?.[networkConfig.defaultNetworkId]?.contractAddress
-
-  if (!contractAddress) {
-    Logger.error(TAG, 'Contract address is not provided in dynamic config')
-    return
+export async function jumpstartLinkHandler(
+  networkId: NetworkId,
+  contractAddress: string,
+  privateKey: string,
+  userAddress: string
+): Promise<Hash[]> {
+  if (networkId !== networkConfig.defaultNetworkId) {
+    throw new Error(`Unsupported network id: ${networkId}`)
```

Comment:
ah, i was wondering why we need to do this, and then i saw that this function is using contract kit 😞 let's leave it for now and make a task to remove contract kit for later

Reply:
ok, here is the task: https://linear.app/valora/issue/RET-1019/[wallet]-refactor-jumstartlinkhandler-to-viem